src/migrations/tasks/20200309204727_user_capabilities.ts

```diff
@@ -0,0 +1,41 @@
+import { makeDomainLogger } from 'back-end/lib/logger';
+import { console as consoleAdapter } from 'back-end/lib/logger/adapters';
+import Knex from 'knex';
+
+const logger = makeDomainLogger(consoleAdapter, 'migrations');
+
+const SKILLS = [
+  'Delivery Management',
+  'Front-End Development',
+  'Back-End Development',
+  'DevOps',
+  'User Experience Design',
+  'User Interface Design',
+  'User Research'
+];
+
+export async function up(connection: Knex): Promise<void> {
+  await connection.schema.alterTable('users', table => {
+    table.specificType('capabilities', 'TEXT[]').defaultTo('{}').notNullable();
+  });
+
+  await connection.schema.raw(`
+  ALTER TABLE "users" \
+  ADD CONSTRAINT "users_capabilities_check" \
+  CHECK (capabilities <@ ARRAY['${Object.values(SKILLS).join('\',\'')}']) \
```

Comment:
IMO don't need to check the capability is valid in the DB, just use validation functions to do the check

Reply:
Agreed, removed.